src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,143 @@
+            isActive: Boolean(syncPeople),
+            onToggle: () => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.SYNCE_PEOPLE, !syncPeople),
+            pendingAction: pendingFields?.syncPeople,
+        },
+        {
+            title: translate('workspace.qbo.advancedConfig.createEntities'),
+            subTitle: translate('workspace.qbo.advancedConfig.createEntitiesDescription'),
+            isActive: Boolean(autoCreateVendor),
+            onToggle: () => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR, !autoCreateVendor),
+            pendingAction: pendingFields?.autoCreateVendor,
+        },
+    ];
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policyID}>
+                <FeatureEnabledAccessOrNotFoundWrapper
+                    policyID={policyID}
+                    featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+                >
+                    <ScreenWrapper
+                        includeSafeAreaPaddingBottom={false}
+                        shouldEnableMaxHeight
+                        testID={QuickbooksAdvancedPage.displayName}
+                    >
+                        <HeaderWithBackButton title={translate('workspace.qbo.advancedConfig.advanced')} />
+
+                        <ScrollView contentContainerStyle={[styles.pb2, styles.ph5]}>
+                            {qboSyncToggleSettings.map((item) => (
+                                <ToggleSettingOptionRow
+                                    key={item.title}
+                                    title={item.title}
+                                    subtitle={item.subTitle}
+                                    shouldPlaceSubtitleBelowSwitch
+                                    wrapperStyle={styles.mv3}
+                                    isActive={item.isActive}
+                                    onToggle={item.onToggle}
+                                    pendingAction={item.pendingAction}
+                                />
+                            ))}
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.reimbursedReports')}
+                                subtitle={translate('workspace.qbo.advancedConfig.reimbursedReportsDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                pendingAction={pendingFields?.reimbursementAccountID && pendingFields?.collectionAccountID}
```

Comment:
```suggestion
                                pendingAction={pendingFields?.reimbursementAccountID || pendingFields?.collectionAccountID}
```
We'd have a pending action if any of the fields is in a pending state